src/Facility.CodeGen.AspNet/AspNetGenerator.cs

```diff
@@ -36,16 +41,25 @@ protected override CodeGenOutput GenerateOutputCore(ServiceInfo serviceInfo)
 			{
 				CSharpUtility.WriteFileHeader(code, GeneratorName);
 
-				List<string> usings = new List<string>
-				{
-					"System",
-					"System.Net.Http",
-					"System.Threading",
-					"System.Threading.Tasks",
-					"System.Web.Http",
-					"Facility.Core",
-					apiNamespaceName
-				};
+				List<string> usings = Target == TargetFramework.Core  ?
```

Comment:
Change this to add to `usings` whatever namespaces are not common to both targets (`System.Web.Http` vs. `Microsoft.AspNetCore.Mvc`). I don't remember why `System` and `Facility.Core` were added but if they are needed for `webapi` in some scenarios, they will likely be needed by `core` in the same scenarios.